internal/olm/client.go

```diff
@@ -386,7 +265,7 @@ func filterResources(resources []unstructured.Unstructured, filter func(unstruct
 	return filtered
 }
 
-func (c Client) getSubscriptionCSV(ctx context.Context, subKey types.NamespacedName) (*types.NamespacedName, error) {
+func (c Client) getSubscriptionCSV(ctx context.Context, subKey types.NamespacedName) (types.NamespacedName, error) {
 	var csvKey *types.NamespacedName
```

Comment:
If we're returning a non-pointer, we might as well change this to:

```suggestion
	var csvKey types.NamespacedName
```

And then update line 280 to use a non-pointer when it sets `csvKey`